apps/lfx-one/src/app/modules/dashboards/org/org-meetings/components/org-upcoming-meetings/org-upcoming-meetings.component.html

```
@@ -0,0 +1,205 @@
+                <span class="inline-flex items-center gap-1.5 text-blue-600">
+                  <i class="fa-light fa-circle-check text-xs" aria-hidden="true"></i> {{ meeting.rsvpTally.yes }} Yes
+                </span>
+                <span class="inline-flex items-center gap-1.5 text-amber-600">
+                  <i class="fa-light fa-circle-question text-xs" aria-hidden="true"></i> {{ meeting.rsvpTally.maybe }} Maybe
+                </span>
+                <span class="inline-flex items-center gap-1.5 text-red-500">
+                  <i class="fa-light fa-circle-xmark text-xs" aria-hidden="true"></i> {{ meeting.rsvpTally.no }} No
+                </span>
+              </div>
+
+              @if (meeting.orgInvitees.length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)